linux/user-and-file-management/linux-intro/linux-common-terminology.md

```diff
@@ -26,6 +26,10 @@ The **package manager** removes the tedious process of installing or uninstallin
 
 > 💡 As with all the skills in the app, if you want to access more advanced or specific content at any time, **feel free to use the *Skill Map* tab** to jump to the workouts most relevant for you.
 
+> 💬 Why are you interested in learning Linux?
```

Comment:
this sounds a bit weird to me, maybe we can say "learning about Linux"?

cc @nem035 @Loopiezlol 

```suggestion
> 💬 Why are you interested in learning about Linux?
```

Reply:
totally agree